SPECS-EXTENDED/hunspell/hunspell.spec

```diff
@@ -120,6 +122,10 @@ mkdir $RPM_BUILD_ROOT/%{_datadir}/myspell
 %{_mandir}/man5/hunspell.5.gz
 
 %changelog
+* Wed Jun 22 2022 Riken Maharjan <rmaharjan@microsoft.com> - 1.7.0-7
+- Patch for the print format issue.
+- License verified
```

Comment:
Thanks for doing that. This spec is a bit more tricky than most due to the number of licenses. It looks like `license.myspell` mentions BSD, so we'd need to add that license to the `LICENSE` tag at the top of the spec.

One more thing we do while verifying the license is placing all license-related files under the `%license` macro (not under `%doc`!) inside the `%files` sections. In the case of this spec this means we should add `%license COPYING COPYING.LESSER COPYING.MPL license.hunspell license.myspell` one line above `%doc` and remove these files from the `%doc` macro.